discord/message.py

```diff
@@ -323,16 +327,33 @@ async def save(
             Saving the attachment failed.
         NotFound
             The attachment was deleted.
+        InvalidArgument
+            Argument `chunksize` is less than 1.
         """
-        data = await self.read(use_cached=use_cached)
+        if chunksize:
```

Comment:
This should make it raise for values of 0
```suggestion
        if chunksize is not None:
```